executive_logic/executive_logic.py

```diff
@@ -1,36 +1,204 @@
-'''
-# startgame (list_of_configurations):
-- apply configuration 
-- start round 
-    - chose_catagory()
-- loop until finished
-
-return results_of_game # total points
-'''
+from board.board import Board
+from score_keeper.score_keeper import ScoreKeeper
+from ui.user_interface import UserInterface
+from wheel.wheel import Wheel, Sector
+from server import GameServer, QueryStatus, Message, MessageType
+from enum import Enum
+import time
+
 MAX_SPINS = 50
 
+
 class ExecutiveLogic:
-    def __init__(self):
-        self.num_spins = 1
-        self.round = 1
-        pass
+    def __init__(self, srv_ip, srv_port):
+        self.board = Board()  # Board object
+        self.score_keeper = ScoreKeeper()  # ScoreKeeper object
+        self.wheel = Wheel(["food", "politics"])  # Wheel object (TODO: These categories should be specified somewhere else, right?)
+        self.ui = UserInterface()  # UI Object
+        self.game_server = GameServer(srv_ip, srv_port, self)  # GameServer object
+
+        self.is_game_running = True  # True if game is ongoing, False otherwise
+        self.num_spins = 0  # Number of times wheel has been spun this round (round ends after MAX_SPINS)
+
+        self.query_status = QueryStatus.STANDBY
+        self.query_response = Message(MessageType.EMPTY, [])
+
+    def run_game(self):
+        """
+        Main game loop. Controls logic through the flow of the entire game.
+        :return:
+        """
+        self.is_game_running = True
+
+        for round_num in [1, 2]:
+            self.board.reset_board(round_num)
```

Comment:
For the bankrupt, does it bankrupt you just for the round? If so, we have to make a function to store the round 1 scores in the scorekeeper and reset the scores for round 2.